sgx/demo/Makefile

```diff
@@ -330,31 +350,27 @@ $(App_Name): $(App_Source_Files) \
              enclave/ecdh_dummy_server.o \
```

Comment:
This is related to an earlier comment, but if we do get rid of the "dummy_server" code, we will need to clean up the Makefile also.